src/components/artwork_details.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+
+export class ArtworkDetails extends React.Component<RelayProps, null> {
+
+  artistLine() {
+    const cultural_maker = this.props.artwork.cultural_maker
+    const artists = this.props.artwork.artists
+
+    if (cultural_maker) {
+      return (<h4>{cultural_maker}</h4>)
+    } else if (artists && artists.length) {
+      let artistsEl = []
+      for (var i=0; i < artists.length; i++) {
```

Comment:
In this case we actually want to know if the element is the last in the array so a for loop makes sense.